src/test/java/teammates/test/cases/util/StringHelperTest.java

```diff
@@ -127,6 +128,33 @@ public void testKeyEncryption() throws InvalidParametersException {
         assertEquals(msg, decrptedMsg);
     }
 
+    private static String encryptWithoutSpecifyingAlgorithmParams(String value) throws Exception {
+
+        SecretKeySpec sks = new SecretKeySpec(StringHelper.hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
+        Cipher cipher = Cipher.getInstance("AES");
+        cipher.init(Cipher.ENCRYPT_MODE, sks, cipher.getParameters());
+        byte[] encrypted = cipher.doFinal(value.getBytes());
+        return StringHelper.byteArrayToHexString(encrypted);
+
+    }
```

Comment:
Place the method _below_ where it is used first. To read code in a top-down manner, lower level methods should be placed below higher level methods.